errors.go

```diff
@@ -37,6 +37,9 @@ var ErrShuttingDown = errors.New("kafka: message received by producer in process
 // ErrMessageTooLarge is returned when the next message to consume is larger than the configured Consumer.Fetch.Max
 var ErrMessageTooLarge = errors.New("kafka: message is larger than Consumer.Fetch.Max")
 
+// ErrMessageTooLarge is returned when a JoinGroup request returns a protocol type that is not supported by sarama.
+var ErrUnknownGroupProtocol = errors.New("kafka: encountered an unknown group protocol")
```

Comment:
This can't happen, the broker selects the protocols supported by all members, and if no such set exists throws a InconsistentGroupProtocol